Frends.IMAP.ReadEmail/README.md

```diff
@@ -22,6 +22,7 @@ https://www.myget.org/F/frends-tasks/api/v2.
 |AcceptAllCerts             |bool                       |Accept all certificates when connecting the host, if true, will accept event invalid certificates. If false, will accept self-signed certificates if the root is untrusted|false|
 |UserName                   |string                     |Account name to login with|emailUser|
 |Password                   |string                     |Account password          |***|
+|ArchiveDirectory           |string                     |General directory for storing attachments|c:/SavedAttachments|
```

Comment:
Actually we do not do docs like this anymore and since you are already here - can you clean up this readme a bit? Here is how we do it today: https://github.com/FrendsPlatform/Frends.AmazonS3/tree/main/Frends.AmazonS3.DownloadObject

So essentially just nuke descriptions of parameters and result and see that it matches with the link above.